examples/lock-app/nrfconnect/README.md

```diff
@@ -391,11 +391,11 @@ To build for the low-power configuration, run the following command with
 _build-target_ replaced with the build target name of the Nordic Semiconductor's
 kit you own (for example `nrf52840dk_nrf52840`):
 
-    $ west build -b build-target -- -DOVERLAY_CONFIG=overlay-low_power.conf
+    $ west build -b build-target -- -DOVERLAY_CONFIG=../../overlay-low_power.conf
```

Comment:
That looks like unintentional change. It was changed back and forth recently and you probably so you probably used the intermediate commit as a base for your changes.

Reply:
Definitely unintended. I will update.

Reply:
Fixed.